aap_chatbot/src/App.test.tsx

```diff
@@ -560,10 +560,11 @@ test("Debug mode test", async () => {
   await renderApp(true);
   await expect.element(page.getByText("granite3-1-8b")).toBeVisible();
   await page.getByText("granite3-1-8b").click();
-  await expect
-    .element(page.getByRole("menuitem", { name: "granite3-8b" }))
-    .toBeVisible();
-  await page.getByRole("menuitem", { name: "granite3-8b" }).click();
+  // Comment out following lines for now since granite3-1-8b is the only choice.
```

Comment:
@TamiTakamiya is this code kept for granite3.2?

Reply:
Yes.  As soon as, 3.2 becomes available, I will un-comment these lines and update them for 3.2.